### PR TITLE
Correction in neural network photo

### PR DESCRIPTION
**Pull request by ArtemioA**
Correction in neural network photo

### CODE REVIEW

**Comment by pescap** [comment]
Nice!

File: docs/source/NeuralNetworkTheory.rst [hunk 1]
```diff
@@ -110,9 +110,11 @@ data.
 .. image:: ~Image/NeuralNetworkbyAndrewNg.png
 (by Andrew Ng, Machine Learning Coursera)
 
-.. image:: ~Image/RedExplicada.png
+.. image:: ~Image/RedExplicadaFinished.png
```
Comment (pescap) [inline docs/source/NeuralNetworkTheory.rst:113]: Could you put the red arrows behind the black texts? They are hiding the text (for example "Layer 2").
And maybe you could add transparency to the red arrows. Thank you!

Comment (pescap) [inline docs/source/NeuralNetworkTheory.rst:113]: could you please change the name for this folder. 
Replace: `~Image` by `img`. Let me know if that is ok for you, thank you!

Comment (ArtemioA) [reply docs/source/NeuralNetworkTheory.rst]: Professor,
No problem!

Comment (ArtemioA) [reply docs/source/NeuralNetworkTheory.rst]: Professor,
No problem!

**Comment by pescap** [comment]
Do not merge yourself!!!

**Pull request merged**